hound.rb

```diff
@@ -0,0 +1 @@
+puts 'long long long long long long long long long long long long long long long long line'
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.